### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by jerone** [orta/vscode-twoslash-queries#40]
Please consider creating an changelog and bundle the an changelog with your extension.

Updating extensions in VSCode is a very hidden process. For people interested in knowing the latest changes, it's difficult to know what changed.

I also couldn't find any [Releases](https://github.com/orta/vscode-twoslash-queries/releases) on GitHub, or any [git tags](https://github.com/orta/vscode-twoslash-queries/tags).

Creating an changelog and bundling it with the package allows users to see all changes under the ['Version history' tab](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-twoslash-queries&ssr=false#version-history) in the Marketplace (for example https://marketplace.visualstudio.com/items/ryu1kn.extension-update-reporter/changelog).

And it allows [Extension Update Reporter](https://marketplace.visualstudio.com/items?itemName=ryu1kn.extension-update-reporter) extension to correctly display your changelog.
![image](https://github.com/user-attachments/assets/e48df7a1-8ef9-41f2-8dce-f0d6c7da4909)

Combining that with GitHub Releases allows users to inspect what changes are part of the extension. It's kinda a security feature too.

**Comment by orta** [comment]
Afraid it's not going to happen, it's not an important enough project for me to do this - sorry

**Comment by jerone** [comment]
Not even git tags? You are already doing [version update commits](https://github.com/orta/vscode-twoslash-queries/commit/d461407faff408be22a776c0c21ffc52799d0a33).

**Comment by orta** [comment]
No, this project is lucky enough to get updates, I'm not giving myself more work
